KA-Flocking/Assets/Scripts/restartGame.cs

```diff
@@ -0,0 +1,24 @@
+﻿//This script is controlling the button.
+//Later on this script will fetch the scene that starts a new game.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+public class restartGame : MonoBehaviour
+{
+    public Text text;
+	public Button yourButton;
+
+		void Start()
+		{
+			//Button btn = yourButton.GetComponent<Button>();
+			yourButton.onClick.AddListener(ResetGame);
+		}
+
+		void ResetGame()
+		{
+		//Here should the scene for the game be loaded!
+		text.text = ("Play again!");
```

Comment:
This file has some weird formatting. Sometime a lot of indenting (basically everywhere), sometimes only a little (line 10) and sometimes non at all (line 20-21) 😁